apps/storybook/.storybook/main.js

```diff
@@ -6,23 +6,8 @@ module.exports = {
   },
   stories: ['../src/**/*.stories.@(tsx|mdx)'],
   addons: [
-    {
-      name: '@storybook/preset-create-react-app',
-      // Override react-scripts package path as it resolves to the wrong path on Linux
-      options: { scriptsPackageName: '../node_modules/react-scripts' },
-    },
+    'storybook-css-modules-preset',
```

Comment:
Storybook doesn't support CSS modules by default... :disappointed:

Reply:
Well, it could be worse than just an addon to add :smile: